package.json

```diff
@@ -91,8 +91,8 @@
   "xo": {
     "esnext": true,
     "ignores": [
-      "test/test.js",
-      "dist/**"
+      "dist/**",
+      "lib/index.js"
```

Comment:
That's weird. I already handle that here: https://github.com/sindresorhus/eslint-config-xo/blob/475b2a3692e230c45319ebe36700c97f96621d62/esnext.js#L16